modelforge/potential/models.py

```diff
@@ -907,6 +907,7 @@ def setup_potential(
         jit=jit,
         jit_neighborlist=False if use_training_mode_neighborlist else True,
     )
+    model.train()
```

Comment:
Since `setup_potential` is used to set up models for interference, why are you are marking it in training mode here?

Reply:
good catch! that was definitly not intended, setting this to eval